### ISSUE TITLE
NeuralStyler AI

### ISSUE THREAD

**Issue by rupeshs** [gafr/chainer-fast-neuralstyle-models#5]
Check out this 
Turn Your Videos/Photos/GIF into Art
http://neuralstyler.com/

*Style videos,gif animation and photos
*No need to upload videos (Offline processing)
*Faster AI styling algorithm
*Extensible styling system(Plugin)

Simply copy the model to the style directory 
edtaonisl.model
edtaonisl-style.jpg

**Comment by 6o6o** [comment]
Wow, impressive. Well done with offline processing. I read that Prisma also decided to go offline with the latest update just yesterday, so it appears to be a trend now :)

I have a few questions. As far as i understand you have packaged Chainer along with all dependencies and Python executables in a virtualenv structure. Would this also work for other Linux distributions, like CentOS or Fedora? In particular, I'm interested in using this on android. Do you think it would work? If not, what modifications would be required to successfully port this to android?

Thanks.

**Comment by rupeshs** [comment]
@6060 Thanks...
NeuralStyler uses Chainer to process .It can process video/photo/gif and I tested only in Ubuntu 14.04 64 bit...it may work on other Linux distributions...Currently I am working to develop windows version :) 
Source code 
https://github.com/rupeshs/neuralstyler

**Comment by DylanAlloy** [comment]
@rupeshs check out py2exe

**Comment by rupeshs** [comment]
@DylanAlloy py2exe won't work because chainer and numpy will be headache, even I tried nuitka (python to C++ compiler) after series of failures I selected virtualenv.

**Comment by DylanAlloy** [comment]
@rupeshs that's a bummer. I've worked with weird apis and it handled it pretty well. did you try pyinstaller with no upx? I know it's probably a long shot but i'm just naming things that surprised me.

**Comment by rupeshs** [comment]
@DylanAlloy Yes,I tried pyinstaller but the same result..chainer dependencies makes pain.

**Comment by rupeshs** [comment]
Winodws version available and more styles  http://neuralstyler.com/
http://neuralstyler.com/styles.html

**Comment by rupeshs** [comment]
NeuralStyler 2 released 
-Adjust style strength
-Video trimmer
-Added option to keep aspect ratio
-Custom resolution
-Added pause functionality
-Added online help documentation
-Option to disable scaling
-Option to preview styled frame
-Drag and drop support
http://neuralstyler.com/
